sportsref/ncaaf/teams.py

```diff
@@ -75,4 +75,28 @@ def sos(self, year):
             print "ERROR: NO SRS FOUND FOR {} in {}".format(self.teamID, year)
             return np.nan
 
-# TODO - BEGIN HERE
+    def get_roster(self, year):
+        """ Returns the roster for a team for the year
+
+        :param year: the year of the roster
+        :return: a dataframe with the player_id, class, and position
+        """
+        url = sportsref.ncaaf.BASE_URL +\
+              '/schools/{}/{}-roster.html'.format(self.teamID, str(year))
+        doc = pq(sportsref.utils.getHTML(url))
+        table = doc('#all_roster')
+        df = sportsref.utils.parseTable(table)
+        return df
+
+def get_all_college_teams():
```

Comment:
I suppose it depends on the use case for this function. I guess there's no harm in leaving this the way it is now; if I change my mind, I may move things around, but for now, this is fine with me.